x-pack/plugins/rollup/public/crud_app/sections/job_list/job_list.js

```diff
@@ -87,7 +87,7 @@ export class JobListUi extends Component {
       defaultMessage: 'Permission error',
     });
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
+      <EuiPageSection alignment="center" color="danger">
```

Comment:
Before:
<img width="820" alt="Screenshot 2023-10-12 at 12 38 56 PM" src="https://github.com/elastic/kibana/assets/5226211/b45e8b71-e5be-4974-b40c-005c2d74ca5e">


After:
<img width="1583" alt="Screenshot 2023-10-12 at 12 39 04 PM" src="https://github.com/elastic/kibana/assets/5226211/259aca1d-f570-42f3-b236-a41a7863517f">

Reply:
Should be fixed by fe70f986a40e830d03ddedc1d31c97fdafd500e6